quickwit/quickwit-cli/src/index.rs

```diff
@@ -1069,12 +1069,33 @@ pub async fn merge_cli(args: MergeArgs) -> anyhow::Result<()> {
             pipeline_ord: 0,
         })
         .await?;
-    let pipeline_handle = indexing_service_mailbox
-        .ask_for_res(DetachIndexingPipeline { pipeline_id })
+    let pipeline_handle: ActorHandle<MergePipeline> = indexing_service_mailbox
+        .ask_for_res(DetachMergePipeline {
+            pipeline_id: MergePipelineId::from(&pipeline_id),
+        })
         .await?;
-    let (pipeline_exit_status, _pipeline_statistics) = pipeline_handle.join().await;
+
+    let mut check_interval = tokio::time::interval(Duration::from_secs(1));
+    loop {
+        check_interval.tick().await;
+
+        let observation = pipeline_handle.observe().await;
+
+        if observation.num_ongoing_merges == 0 {
```

Comment:
Nitpick: we can add an `info` log to give the reason why the merge stopped.

Reply:
Added!